app/helpers/spree/admin/product_catalogs_helper.rb

```diff
@@ -0,0 +1,17 @@
+module Spree
+  module Admin
+    module ProductCatalogsHelper
+      def product_catalog_options(variant, product_catalog)
```

Comment:
If this is being used in an API action, would it be better to move it to its own action and use jbuilder? We would be able to take advantage of fragment caching in that case.

Reply:
this is not being used in an API action, I made this helper to bring all the information we need to call the react component on `engine_storefront` -> https://github.com/geminimvp/engine_storefront/pull/455/files#diff-117ea643f27a672f43375fd2348db2d4R6 and https://github.com/geminimvp/engine_storefront/pull/455/files#diff-5eaede60dc1db18d7a3cb0345a8d4d29R34